src/components/HistoryArchivedTextChange/HistoryArchivedTextChange.tsx

```diff
@@ -0,0 +1,34 @@
+import { nullable } from '@taskany/bricks';
+import { Text } from '@taskany/bricks/harmony';
+
+import { tr } from './HistoryArchivedTextChange.i18n';
+import s from './HistoryArchivedTextChange.module.css';
+
+interface HistoryChangeProps {
+    from?: string | null;
+    to?: string | null;
+}
+
+export const HistoryArchivedTextChange: React.FC<HistoryChangeProps> = ({ from, to }) => {
+    const getTextContent = (text: string) => {
+        return text === 'true' ? tr('in archive') : tr('not in archive');
+    };
+    return (
+        <div className={s.HistoryArchivedTextChange}>
```

Comment:
You can reuse HistoryTagsAndDifficultyTextChange, but rename it to abstract 'PlainTextChange'